### DISCUSSION TITLE
Remove Menu Item "Edit this Page"

### DISCUSSION THREAD

**Discussion by jrief** [django-cms/django-cms#7997] [Polls]
During a conversation with @fsbraun and @jrief we agreed on removing the submenu item `Edit this Page` from the toolbar's `Page` menu item. Reason is that this functionality is duplicated by the button on the right side of the toolbar named `New Draft`.
Second reason is that this button is disabled by default and only enabled when the page is in edit mode anyway. This is very confusing for users of the CMS.

<img width="197" alt="Screenshot 2024-09-13 at 08 53 42" src="https://github.com/user-attachments/assets/29bbd29b-7be2-4e59-8019-00fccdc4f429">

**Comment by fsbraun** [comment]
I support this since versioning, moderation etc. modify the edit and publication cycle, leading to inconsistent menu entries (alias does not have this).

**Comment by MacLake** [comment]
I think nobody uses the menu entry for editing, so remove it.

Btw. of course your question is understandable, but it would also be less confusing to avoid the negation: “Do you agree with this proposal: yes/no”, in case somebody doesn’t read so carefully.

**Comment by fsbraun** [comment]
Implemented with #7999
